README.md

```diff
@@ -1,25 +1,198 @@
+  </a>
+  <a href="https://travis-ci.com/Qiskit/qiskit.org">
+    <img src="https://travis-ci.com/Qiskit/qiskit.org.svg?branch=master" alt="Current CircleCI build status." />
+  </a>
+  <a href="https://github.com/Qiskit/qiskit.org/blob/master/CONTRIBUTING.rst">
+    <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" alt="PRs welcome!" />
+  </a>
+  <a href="https://twitter.com/intent/follow?screen_name=qiskit">
+    <img src="https://img.shields.io/twitter/follow/qiskit.svg?label=Follow%20@qiskit" alt="Follow @qiskit" />
+  </a>
+</p>
 
-This repo is for the website for qiskit.org.
+<h3 align="center">
+  <a href="https://qiskit.org/">Home page</a>
+  <span> · </span>
+  <a href="https://qiskit.org/education/">Education</a>
+  <span> · </span>
+  <a href="https://qiskit.org/documentation/">Documentation</a>
+  <span> · </span>
+  <a href="https://qiskit.org/experiments/">Experiments</a>
+  <span> · </span>
+  <a href="https://qiskit.org/advocates/">Advocates</a>
+  <span> · </span>
+  Support: <a href="https://join.slack.com/t/qiskit/shared_invite/enQtODQ2NTIyOTgwMTQ3LTI0NzM2NzkzZjJhNDgzZjY5MTQzNDY3MGNiZGQzNTNkZTE4Nzg1MjMwMmFjY2UwZTgyNDlmYWQwYmZjMjE1ZTM">Slack</a>
+</h3>
 
+## What’s In This Document
 
-## Build Setup
+- [Live](#%EF%B8%8F-live)
+- [Technology Used](#-technology-used)
+- [Get Up and Running](#-get-up-and-running)
+- [Folder Structure](#-folder-structure)
+- [How to Contribute](#%EF%B8%8F-how-to-contribute)
+- [Available Scripts](#-available-scripts)
+- [Open Backlog](#-open-backlog)
+- [Maintainers](#-maintainers)
 
-``` bash
-# install dependencies
-$ npm install
+## ⚡️ Live
+[Go to Qiskit.org's website](https://qiskit.org/)
 
-# serve with hot reload at localhost:3000
-$ npm run dev
+## 💻 Technology Used
 
-# build for production and launch server
-$ npm run build
-$ npm start
+Qiskit.org is a pre-rendering SPA using [Nuxt.js](https://nuxtjs.org/).
 
-# generate static project
-$ npm run generate
+A **pre-rendering SPA** is a single page application that generates a static markup (HTML) at build time. The user, when entering the web, receives HTML (as if it were a static web) but in the meantime, JS files belonging to the SPA are loaded “hydrating” the web until it's completely dynamic.
+
+**[Nuxt.js](https://nuxtjs.org/)** is the biggest framework on top of **[Vue.js](https://vuejs.org/)** to generate *universal* SPAs. Universal or "isomorphic" apps can be pre-rendering or SSR. Since so far we don't need server functions, our website it's just pre-rendering.
+
+We create and run unit tests using [Jest](https://jestjs.io/), ensure avoiding syntax errors using [ESLint](https://eslint.org/) and automate all these previous tools and deployment using [Trevis](https://travis-ci.org/).
+
+With this technology we want to **achieve**:
+- Use a component based framework like Vue that allow us to reuse part of the UI code in different parts of the application.
```

Comment:
I'd add separation between content edition and development concerns.

Reply:
Fixed in the last commit 😊 thanks!